utils/lib/rr_graph/points.py

```diff
@@ -1147,15 +1163,3 @@ def print_tracks(ret):
 
     print('x = {}'.format(x_tracks))
     print('y = {}'.format(y_tracks))
-
```

Comment:
Why is this removed?

Reply:
It's not used and I think having removing main() methods from library type modules is a good step toward having them as proper packages.

Reply:
I normally find it useful to be able to quickly test a whole file by running the library file...